src/components/hint/index.js

```diff
@@ -0,0 +1,335 @@
+// TODO: Add support to custom hint rendering
+// TODO: Add animation
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import {View as AnimatableView} from 'react-native-animatable';
```

Comment:
what is the purpose of this?

Reply:
What do u mean? For adding entrance animation to the hint.

Reply:
I mean why use AnimatableView instead of Animatable.View? Is the reason is that it is better to import just the View instead the * as Animatable?

Reply:
yea, the `AnimatableView` is just an alias for `Animatable.View`. 
I didn't want to import everything.

Reply:
That's what I thought. We should use this in other imports of Animatable...

Reply:
agree, can we merge? or is there anything else?